lib/puppet/provider/service/debian.rb

```diff
@@ -16,7 +16,8 @@
   # is resolved.
   commands :invoke_rc => "/usr/sbin/invoke-rc.d"
 
```

Comment:
Why not add "confine :false => Puppet::FileSystem.exist?("/run/systemd/system")" here?